kubetest/e2e.go

```diff
@@ -506,16 +518,21 @@ func KubemarkTest(dump string, o options) error {
 	})
 
 	// Start new run
-	if popN, err := pushEnv("NUM_NODES", o.kubemarkNodes); err != nil {
+	if popN, err := pushEnv("NUM_NODES", nodes); err != nil {
 		return err
 	} else {
 		defer popN()
 	}
-	if popM, err := pushEnv("MASTER_SIZE", o.kubemarkMasterSize); err != nil {
+	if popM, err := pushEnv("MASTER_SIZE", masterSize); err != nil {
```

Comment:
Cool, I can send a PR for this.

> My primary/only goal here is to move config that lives in global env state and refactor it into flags.
> I don't have the time/expertise required to refactor everyone's config usage.

Since you were anyway moving this config 'kubemarkMasterSize' that's purely kubemark-specific, I was suggesting it. Would've saved you some extra code in the PR :)